.changeset/friendly-glasses-sleep.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-nomad-backend': patch
+'@backstage/plugin-nomad': patch
```

Comment:
```suggestion
'@backstage/plugin-nomad-backend': minor
'@backstage/plugin-nomad': minor
```
as this is the first version let's make this minor so we get 0.1.0 as the first release.